docs/source/en/optimization/fp16.md

```diff
@@ -174,11 +174,18 @@ Feel free to open an issue if dynamic compilation doesn't work as expected for a
 
 ### Regional compilation
 
+[Regional compilation](https://docs.pytorch.org/tutorials/recipes/regional_compilation.html) trims cold-start latency by only compiling the *small and frequently-repeated block(s)* of a model - typically a transformer layer - and enables reusing compiled artifacts for every subsequent occurrence.
```

Comment:
Actually regional compilation is preferred this way (with `compile_repeated_blocks()`). So, I would keep it as is.